test/manual/atlas-data-lake-testing/atlas_data_lake_testing.spec.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const path = require('path');
+const {
+  TestRunnerContext,
+  gatherTestSuites,
+  generateTopologyTests
+} = require('../../tools/spec-runner');
+
+describe('Atlas Data Lake - spec', function () {
+  const testContext = new TestRunnerContext({
+    skipPrepareDatabase: true,
```

Comment:
The spec says that the runner setup is 

> The same as the CRUD Spec Test Runner Implementation, with one notable differences: the test runner for Atlas Data Lake Testing MUST NOT drop the collection and/or database under test. In contrast to other CRUD tests, which insert their own data fixtures into an empty collection, data for these tests is specified in the mongohoused configuration file.

Does this skipPrepareDatabase flag fully take care of that requirement?

Reply:
I believe so: prepareDatabaseForSuite in https://github.com/mongodb/node-mongodb-native/blob/NODE-3795/fix-no-authProvider-for-default-error/test/tools/spec-runner/index.js#L201 is where the flag is used and you can see it returns early before any set up logic that would drop dbs/collections